Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -6531,7 +6531,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if ! cmp -s \"$SRCROOT/Cartfile.resolved\" \"$SRCROOT/Carthage/Cartfile.resolved_at_last_bootstrap\"\nthen\ncarthage bootstrap --platform iOS\nplutil -replace CFBundleVersion -string 4.0.0 \"$SRCROOT/Carthage/Build/iOS/SDWebImage.framework/Info.plist\"\nmkdir -p \"$SRCROOT/Carthage\"\ncp \"$SRCROOT/Cartfile.resolved\" \"$SRCROOT/Carthage/Cartfile.resolved_at_last_bootstrap\"\nfi";
+			shellScript = "if ! cmp -s \"$SRCROOT/Cartfile.resolved\" \"$SRCROOT/Carthage/Cartfile.resolved_at_last_bootstrap\"\nthen\ncarthage bootstrap --platform iOS 2> /dev/null\nplutil -replace CFBundleVersion -string 4.0.0 \"$SRCROOT/Carthage/Build/iOS/SDWebImage.framework/Info.plist\"\nmkdir -p \"$SRCROOT/Carthage\"\ncp \"$SRCROOT/Cartfile.resolved\" \"$SRCROOT/Carthage/Cartfile.resolved_at_last_bootstrap\"\nfi";
```

Comment:
Outputs to stderr by the script appear in Xcode as build errors - it then confused Xcode when it got errors but still returned 0. The only true failure should be if carthage doesn't return 0.